src/averageAges.js

```diff
@@ -14,12 +14,20 @@
  *
  * @return {number}
  */
+function getAverageAge(peopleFiltered) {
+  const personAge = peopleFiltered.map(person => person.died - person.born);
+  const averageAge = personAge.reduce((a, b) => a + b) / personAge.length;
+
+  return Math.round(averageAge * 100) / 100;
+}
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const male = century
+    ? people.filter(
+      person => person.sex === 'm' && Math.ceil(person.died / 100) === century)
+    : people.filter(person => person.sex === 'm');
```

Comment:
Краще написати один фільтр і в ньому вибирати колбек
```suggestion
  const male = people.filter(
     сentury 
       ?  person => person.sex === 'm' && Math.ceil(person.died / 100) === century
       : person => person.sex === 'm'
  );
```